common/eth2/src/lighthouse/sync_committee_rewards.rs

```diff
@@ -0,0 +1,23 @@
+use serde::{Deserialize, Serialize};
+
+// Details about the rewards paid to sync committee members for attesting headers
+// All rewards in GWei
+
+#[derive(Debug, PartialEq, Clone, Serialize, Deserialize)]
+pub struct SyncCommitteeRewards {
+
+    pub execution_optimistic: Option<bool>,
+
+    pub finalized: Option<bool>,
+
+    pub data: Option<Vec<SyncCommitteeReward>> 
+}
+
+#[derive(Debug, PartialEq, Clone, Serialize, Deserialize)]
+pub struct SyncCommitteeReward {
+
+    pub validator_index: u64,
+    // sync committee reward in gwei for the validator
+    pub reward: i64,
```

Comment:
This should probably also be quoted but we currently don't have anything for `i64`s, and we're in the process of relocating the serde_utils crate. For simplicity lets leave it as-is and I'll add something later.